test/functional/hosts_controller_test.rb

```diff
@@ -878,16 +878,20 @@ class Host::Valid < Host::Managed; end
   end
 
   test 'template_used returns templates with interfaces' do
+    ActiveRecord::Base.any_instance.expects(:destroy).never
```

Comment:
well even if it wouldn't fail it's probably safer to stub just before xhr, thanks